lib/hydra/works/services/generic_file/add_original_file.rb

```diff
@@ -1,10 +1,12 @@
 module Hydra::Works
   class AddOriginalFile
 
-    # Adds the original file to the work
+    # Convenience service for attaching original file
+    # @see Hydra::Works::AddFileToGenericFile 
 
-    def self.call(object, path, replace=false)
-      Hydra::Works::AddFileToGenericFile.call(object, path, ::RDF::URI("http://pcdm.org/OriginalFile"), replace)
+    def self.call(object, file, options={})
```

Comment:
Should the comment block from AddFileToGenericFile be duplicated here?  I was unsure if the `@see` tag would be sufficient. Should it read `# For parameter descriptions, @see Hydra::Works::AddFileToGenericFile` or just copy paste the entire comment block from AddFileToGenericFile?